src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts

```diff
@@ -59,25 +65,29 @@ export class QuestionListComponent implements OnInit, OnDestroy {
   public goToNextQuestionStatus = true;
   previewBtnVisibility = true;
   visibility: any;
-  @ViewChild('resourceTtlTextarea') resourceTtlTextarea: ElementRef;
+  telemetryImpression: any;
+  public telemetryPageId = 'question-list';
 
   constructor(
-    private configService: ConfigService, private userService: UserService,
+    public configService: ConfigService, public userService: UserService,
```

Comment:
@vaibhavbhuva What is the reason to make service as public?

Reply:
@cksachdev We are using this service in the DOM for telemetry. So for that, we have marked it as public.